tests/strands/tools/mcp/test_mcp_from_config.py

```diff
@@ -0,0 +1,200 @@
+import json
+from unittest.mock import patch
+
+import pytest
+
+from strands.tools.mcp.mcp_from_config import (
+    HTTPTransportConfig,
+    MCPServerConfig,
+    MCPTransportType,
+    SSETransportConfig,
+    StdioTransportConfig,
+    _infer_transport_from_url,
+)
+
+
+def create_config_file(tmp_path, config_data):
+    """Helper to create config file."""
+    config_path = tmp_path / "config.json"
+    with open(config_path, "w") as f:
+        json.dump(config_data, f)
+    return str(config_path)
+
+
+class TestMCPTransportType:
```

Comment:
Switch to top-level methods for tests; AFAIK that's the standard throughout the SDK